### PR TITLE
Adding an actual index to main node module

### PR DESCRIPTION
**Pull request by twtaylorbitgo**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -234,16 +235,22 @@ module.exports = {
...
     signMultisigTransaction,
     multisigAddress,
     ERROR_MULTISIG_BAD_SENDER,
+
+    Multisig: multisig,
+    Transaction: txnBuilder,
+    Address: address,
+    Encoding: encoding,
+    Seed,
+    Mnemonic,
```
Comment (CapnMigraine) [inline src/main.js]: `Mnemonic: mnemonic`

File: src/main.js [hunk 2]
```diff
@@ -33,11 +34,11 @@ function generateAccount() {
...
-    const keys = nacl.keyPairFromSeed(seed);
-    return {
-        addr: address.encode(keys.publicKey),
-        sk: keys.secretKey
-    };
+  const keys = nacl.keyPairFromSeed(seed);
+  return {
+      addr: address.encode(keys.publicKey),
+      sk: keys.secretKey
+  };
```
Comment (CapnMigraine) [inline src/main.js]: are these indentation changes intentional? the lib uses 4 spaces everywhere. I think it's better to leave anything we don't need to change as-is so we have to deal with less potential rebase conflicts if we have to re-sync the repo.

File: src/main.js [hunk 3]
```diff
@@ -46,7 +47,7 @@ function generateAccountFromSeed(seed) {
  * @returns {Boolean} true if valid, false otherwise
  */
 function isValidSeed(seed) {
-    return Seed.isValidSeed(seed);
+  return Seed.isValidSeed(seed);
```
Comment (CapnMigraine) [inline src/main.js]: same as above

**Review by CapnMigraine — CHANGES_REQUESTED** [review]

File: src/main.js (hunk 2 above)
Comment (twtaylorbitgo) [reply src/main.js]: not really, can change it back to 4

File: src/main.js (hunk 1 above)
Comment (twtaylorbitgo) [reply src/main.js]: woops, ack

**Review by CapnMigraine — APPROVED** [review]
lgtm

**Pull request merged**